index.html

```diff
@@ -0,0 +1,41 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+</head>
+<body>
+  <h1>Moyo header</h1>
+  <header>
+    <nav class="pages menu1200px">
+      <div class="page0"></div>
+      <a class="page is-active" href="#page1">APPLE</a>
+      <a class="page" href="#page2">SAMSUNG</a>
+      <a class="page" href="#page3">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
+      <a class="page" href="#page4" data-qa="hover">НОУТБУКИ И КОМПЬЮТЕРЫ</a>
+      <a class="page" href="#page5">ГАДЖЕТЫ</a>
+      <a class="page" href="#page6">ПЛАНШЕТЫ</a>
+      <a class="page" href="#page7">ФОТО</a>
+      <a class="page page8" href="#page8">ВИДЕО</a>
+    </nav>
+    <nav class="pages menu1024px">
+      <div class="page page0"></div>
+      <a class="page is-active" href="#page1">APPLE</a>
+      <a class="page" href="#page2">SAMSUNG</a>
```

Comment:
```suggestion
      <a class="page" href="#">SAMSUNG</a>
```
Для всех ссылок этого будет достаточно